python/datafusion/tests/test_context.py

```diff
@@ -158,13 +158,43 @@ def test_from_arrow_table(ctx):
     assert df.collect()[0].num_rows == 3
 
 
+@pytest.mark.skip
```

Comment:
```suggestion
```

`pytest` already doesn't consider this a test because it doesn't start with `test_`.

https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery